test/visual/backgrounds.js

```diff
@@ -91,7 +91,10 @@ async function testBackground(t, name, effect) {
   ['ripples_random'],
   ['squiggles'],
   ['growing_stars'],
-  ['paint_drip']
+  ['paint_drip'],
+  ['blooming_petals', 'vintage'],
+  ['clouds', 'neon'],
+  // ['frosted_grid', 'vintage'], // anchor points are randomly generated, so this test will fail
```

Comment:
Is it worth having a test-specific code path to avoid randomness, so that we can get coverage on these?

Reply:
Good suggestion! Yes - there is actually an already defined `randomNumber` function in `Effects.js` which uses `p5.random` instead of `Math.random`. This allows us to use `p5.randomSeed(0)` in the test before the background tests are run for consistent images. Thanks!

Reply:
I expanded test coverage of effects to include 'smiling-poop' and 'hearts_red' since they both used random integers as well.